src/thinkspace/client/thinkspace-builder-rat/addon/components/content/sections/trat.coffee

```diff
@@ -0,0 +1,31 @@
+import ember         from 'ember'
+import base          from 'thinkspace-base/components/base'
+
+###
+# # irat.coffee
+- Type: **Component**
+- Package: **ethinkspace-builder-rat**
+###
+export default base.extend
+  # classNameBindings: ['has_questions::is-readonly']
+
+  model: null
+  type:  'trat'
+
+  manager: ember.inject.service()
+
+  question_items: ember.computed.reads 'step.trat_question_items'
+
+  has_questions: ember.computed.notEmpty 'question_items'
+
+  display_questions: ember.computed.not 'sync_assessments'
+  sync_checked:      ember.computed.reads 'sync_assessments'
+
+  actions:
+    create: ->
+      manager = @get('manager')
+      manager.add_question_item(@get('type'))
+
+    toggle_assessment_sync: (val) -> 
+      val = if val=='true' then false else true
```

Comment:
Could the `val` be passed into the component? If so, you could just do `@sendAction(@toggleProperty('val'))` - would make it a little easier to understand.